test/unit/test.py

```diff
@@ -2491,6 +2492,44 @@ def test_login(self):
             self.assertFalse('Start' in response.data)
 
     # in TestApp
+    @patch('chime.view_functions.AUTH_CHECK_LIFESPAN', new=1.0)
+    def test_login_timeout(self):
+        ''' Check basic log in / log out flow with auth check lifespan.
+        '''
+        response = self.test_client.get('/')
+        self.assertFalse('Start' in response.data)
+
+        with HTTMock(self.mock_persona_verify_erica):
+            response = self.test_client.post('/sign-in', data={'email': 'erica@example.com'})
+            self.assertEqual(response.status_code, 200)
+
+        with HTTMock(self.auth_csv_example_allowed):
+            response = self.test_client.get('/')
+            self.assertTrue('Start' in response.data)
+
+        with HTTMock(self.mock_exception):
```

Comment:
I struggled to follow this. Maybe be explicit that the mock_exception is an exception only on auth check? It'd  be even clearer just to stub out the auth method and check whether it was called; using an exception to check that confuses the issue some.